app/component/map/tile-layer/TileContainer.js

```diff
@@ -231,22 +238,16 @@ class TileContainer {
               secondY = featureY - width;
             }
           }
-        }
-        let dist = Math.sqrt(
-          (localPoint[0] - featureX) ** 2 + (localPoint[1] - featureY) ** 2,
-        );
-        if (isCombo) {
-          dist = Math.min(
-            dist,
-            Math.sqrt(
-              (localPoint[0] - featureX) ** 2 + (localPoint[1] - secondY) ** 2,
-            ),
+        } */
+
+        return some(g, p => {
+          const dist = Math.sqrt(
+            (localPoint[0] - p.x / this.ratio) ** 2 +
+              (localPoint[1] - p.y / this.ratio) ** 2,
           );
-        }
-        if (dist < 22 * this.scaleratio) {
-          return true;
-        }
-        return false;
+
+          return dist < 22 * this.scaleratio;
+        });
       });
```

Comment:
@leonardehrenfried I really don't know how this works and how I can get things merged with master branch here. Can you please help me here or give a suggestion?

Reply:
The code before always assumed hat a feature is always a point however in this case we want to make the entire polyline clickable. That's what this code is about.

Reply:
Are you seeing an error?

Reply:
Yes, I got that. But there are some changes here on next branch concerning `collision check for stops and citybike stations` and I am not sure how to get this together with polyline collision checks. But I will try.

Reply:
I see, i will take a look.